docs/csharp/language-reference/operators/null-parameter-check.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "!! (null validation) operator - C# reference"
+description: "Learn about simplified null-parameter checks. This operator instructs the compiler to add runtime checks that the argument used for a parameter isn't null."
+ms.date: 03/30/2022
+helpviewer_keywords:
```

Comment:
f1 not yet added.

Reply:
I just handed push that update yet. It is now.